__tests__/cli.ts

```diff
@@ -12,9 +12,12 @@ function exec(
   options: { cwd: string },
 ): Promise<{ exitCode: number | null; stdout: string }> {
   return new Promise((resolve) => {
-    const process = childProcess.exec(command, options, (_error, stdout) => {
-      resolve({ exitCode: process.exitCode, stdout });
-    });
+    try {
+      const buffer = childProcess.execSync(command, options);
```

Comment:
I had purposely tried not to use the `...Sync` versions of anything, for no other reason than my own personal preference.  If it was causing issues then I'm happy for it to change to sync, but I'm still not sure why it's necessary?

Reply:
Is the behavior when being used sync or async?

Reply:
I think the focus should be testing the code the way it will behave or execute in a real situation the best we can